tests/ballerina-unit-test/src/test/resources/test-src/compression/compression.bal

```diff
@@ -1,39 +1,46 @@
 import ballerina/internal;
 
-function decompressFile(string src, string destDir) returns error? {
+function decompressFile(string src, string destDir) returns internal:CompressionError? {
     internal:Path srcPath = new(src);
     internal:Path dstPath = new(destDir);
     var result = internal:decompress(srcPath, dstPath);
```

Comment:
Since we are not processing the result, shall we just return the value got from decompress like
``return internal:decompress(srcPath, dstPath);``